Build error with latest release of Sweetalert

Hello,
with the latest release of the sweetalert core library the build process encounters a problem of typization:

`X [ERROR] TS2749: 'Swal__default.DismissReason' refers to a value, but is being used as a type here. Did you mean 'typeof Swal__default.DismissReason'? [plugin angular-compiler]
 
    node_modules/@sweetalert2/ngx-sweetalert2/index.d.ts:232:35:
      232 │ ...y dismiss: EventEmitter<Swal__default.DismissReason | undefined>;
          ╵                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 
X [ERROR] TS2749: 'Swal__default.DismissReason' refers to a value, but is being used as a type here. Did you mean 'typeof Swal__default.DismissReason'? [plugin angular-compiler]
 
    node_modules/@sweetalert2/ngx-sweetalert2/index.d.ts:362:35:
      362 │ ...y dismiss: EventEmitter<Swal__default.DismissReason | undefined>;`

The installed versions of the packages are:
```
"sweetalert2": "11.26.2",
"@sweetalert2/ngx-sweetalert2": "14.0.0",
```

Please fix the type changes.
I think it was originated here:
[Commit c5af0cf](https://github.com/sweetalert2/sweetalert2/commit/c5af0cf40e5e1c81d9f2d91ae6ee3e7541d33c77) in "sweetalert2": "11.26.2"

Thanks in advance

--- a/projects/ngx-sweetalert2/src/lib/swal.component.ts
+++ b/projects/ngx-sweetalert2/src/lib/swal.component.ts
@@ -1,5 +1,17 @@
-import { AfterViewInit, ChangeDetectionStrategy, Component, EventEmitter, Input, OnChanges, OnDestroy, OnInit, Output, SimpleChanges, inject } from "@angular/core";
-import Swal, { SweetAlertOptions, SweetAlertResult, SweetAlertUpdatableParameters } from "sweetalert2";
+import {
+    AfterViewInit,
+    ChangeDetectionStrategy,
+    Component,
+    EventEmitter,
+    Input,
+    OnChanges,
+    OnDestroy,
+    OnInit,
+    Output,
+    SimpleChanges,
+    inject,
+} from "@angular/core";
+import { SweetAlertOptions, SweetAlertResult, SweetAlertUpdatableParameters } from "sweetalert2";
 import { dismissOnDestroyToken, fireOnInitToken } from "./di";
 import * as events from "./swal-events";
 import { SweetAlert2LoaderService } from "./sweetalert2-loader.service";
@@ -256,13 +268,13 @@ export class SwalComponent implements OnInit, AfterViewInit, OnChanges, OnDestro
      * Example:
      *     <swal (dismiss)="handleDismiss($event)"></swal>
      *
-     *     public handleDismiss(reason: DismissReason | undefined): void {
+     *     public handleDismiss(reason: SweetAlertResult["dismiss"] | undefined): void {
      *         // reason can be 'cancel', 'overlay', 'close', 'timer' or undefined.
      *         // ... do something
      *     }
      */
     @Output()
-    public readonly dismiss = new EventEmitter<Swal.DismissReason | undefined>();
+    public readonly dismiss = new EventEmitter<SweetAlertResult["dismiss"] | undefined>();
 
     /**
      * This Set retains the properties that have been changed from @Inputs, so we can know precisely

--- a/projects/ngx-sweetalert2/src/lib/swal.directive.ts
+++ b/projects/ngx-sweetalert2/src/lib/swal.directive.ts
@@ -1,7 +1,19 @@
-import { ComponentFactoryResolver, ComponentRef, Directive, EventEmitter, HostListener, Input, OnDestroy, OnInit, Output, ViewContainerRef, inject } from "@angular/core";
+import {
+    ComponentFactoryResolver,
+    ComponentRef,
+    Directive,
+    EventEmitter,
+    HostListener,
+    Input,
+    OnDestroy,
+    OnInit,
+    Output,
+    ViewContainerRef,
+    inject,
+} from "@angular/core";
 import { Subject } from "rxjs";
 import { takeUntil } from "rxjs/operators";
-import Swal, { SweetAlertArrayOptions, SweetAlertOptions } from "sweetalert2";
+import { SweetAlertArrayOptions, SweetAlertOptions, SweetAlertResult } from "sweetalert2";
 import { SwalComponent } from "./swal.component";
 
 /**
@@ -94,7 +106,7 @@ export class SwalDirective implements OnInit, OnDestroy {
      *     }
      */
     @Output()
-    public readonly dismiss = new EventEmitter<Swal.DismissReason | undefined>();
+    public readonly dismiss = new EventEmitter<SweetAlertResult["dismiss"] | undefined>();
 
     /**
      * When the user does not provides a SwalComponent instance, we create it on-the-fly and assign the plain-object

